logstash-input-http_poller.gemspec

```diff
@@ -17,13 +17,12 @@ Gem::Specification.new do |s|
   s.metadata = { "logstash_plugin" => "true", "logstash_group" => "input" }
 
   # Gem dependencies
-  s.add_runtime_dependency "logstash-core", ">= 2.0.0", "< 3.0.0"
+  s.add_runtime_dependency "logstash-core", "2.0.0.snapshot2"
```

Comment:
We have to use the explicit form:
```
"2.0.0.snapshot2"
``
Because this "2.0.0.snapshot" doesn't work in the gemspec.


````
Bundler could not find compatible versions for gem "logstash-core":
  In Gemfile:
    logstash-input-stdin (>= 0) java depends on
      logstash-core (= 2.0.0.snapshot) java
Could not find gem 'logstash-core (= 2.0.0.snapshot) java', which is required by
gem 'logstash-input-stdin (>= 0) java', in any of the sources.
```

So.. this mean that if we release another snapshot we have to update and publish all the gems.